.babelrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Are we using babel? I didn't see it on the dependencies (I don't know if it's needed for ts)

Reply:
No we're not. The base I've based on had that file and I forgot to remove it.